osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/infra/jms/ws/messageprocessors/ActualPowerQualityRequestMessageProcessor.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.domain.smartmetering.infra.jms.ws.messageprocessors;
+
+import org.opensmartgridplatform.adapter.domain.smartmetering.application.services.MonitoringService;
+import org.opensmartgridplatform.adapter.domain.smartmetering.infra.jms.BaseRequestMessageProcessor;
+import org.opensmartgridplatform.domain.core.valueobjects.smartmetering.ActualPowerQualityRequest;
+import org.opensmartgridplatform.shared.exceptionhandling.FunctionalException;
+import org.opensmartgridplatform.shared.infra.jms.DeviceMessageMetadata;
+import org.opensmartgridplatform.shared.infra.jms.MessageProcessorMap;
+import org.opensmartgridplatform.shared.infra.jms.MessageType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ActualPowerQualityRequestMessageProcessor extends BaseRequestMessageProcessor {
+
+    @Autowired
+    @Qualifier("domainSmartMeteringMonitoringService")
+    private MonitoringService monitoringService;
+
+    @Autowired
+    public ActualPowerQualityRequestMessageProcessor(
```

Comment:
try to remove all the qualifiers and add all construction injection, Autowired can be omitted.

Reply:
Not modifying this implementation. Possibly modifying this after discussion in broader scope.